docs/pages/heading.js

```diff
@@ -377,7 +379,10 @@ export default function DocsPage({ generatedDocGen }: {| generatedDocGen: DocGen
 `}
           />
         </MainSection.Subsection>
-        <MainSection.Subsection title="Alignment">
+        <MainSection.Subsection
+          title="Alignment"
+          description="Use `align` to adjust the positioning of text within wrapper elements."
```

Comment:
s/"wrapper"/"container"

Reply:
Not sure I follow what this means.

Reply:
Sorry, [`sed` syntax](https://en.wikipedia.org/wiki/Sed#Substitution_command): I'm suggesting that we replace "wrapper" with "container", which better describes the parent element.

Reply:
Oh, nice! TIL! Thanks and fixed!